code/modules/mob/living/carbon/xenomorph/Castes/Sentinel.dm

```diff
@@ -173,15 +173,15 @@
 	if(!check_state())
 		return
 
-	if(world.time < last_neurotoxin_sting + NEUROTOXIN_STING_COOLDOWN) //Sure, let's use this.
-		to_chat(src, "<span class='xenowarning'>You are not ready to use the sting again. It will be ready in [(last_neurotoxin_sting + NEUROTOXIN_STING_COOLDOWN - world.time) * 0.1] seconds.</span>")
+	if(world.time < last_neurotoxin_sting + SENTINEL_STING_COOLDOWN) //Sure, let's use this.
+		to_chat(src, "<span class='xenowarning'>You are not ready to use the sting again. It will be ready in [(last_neurotoxin_sting + SENTINEL_STING_COOLDOWN - world.time) * 0.1] seconds.</span>")
 		return
 
 	if(stagger)
 		to_chat(src, "<span class='xenowarning'>You try to sting but are too disoriented!</span>")
 		return
 
-	if(!istype(H) || isXeno(H) || isrobot(H))
+	if(!istype(H) || isXeno(H) || isrobot(H) || isSynth(H) || H.stat == DEAD)
```

Comment:
At this point redefined a `can_sting_xenotox()` proc is better. Could be re-used for any xenotoxin injection of abilities. Way too many `istype()` checks.